tests/specs/sass-spec-exclude.txt

```diff
@@ -637,6 +701,8 @@ core_functions/selector/unify/simple/universal/and_universal/empty/and_explicit
 core_functions/selector/unify/simple/universal/and_universal/explicit/and_any
 core_functions/selector/unify/simple/universal/and_universal/explicit/and_default
 core_functions/selector/unify/simple/universal/and_universal/explicit/and_empty
+core_functions/string/index/combining_character
+core_functions/string/index/double_width_character
```

Comment:
These 2 failed specs are actually specs that I contributed myself upstream to cover a potential issue in our code (and this proves that we are indeed affected).